src/pilot/inspect.patch

```diff
@@ -1,25 +1,18 @@
---- inspect.py	2019-03-18 04:26:06.117996201 -0400
-+++ inspect_latest.py	2019-03-18 04:36:58.007653944 -0400
-@@ -119,10 +119,17 @@
-                       {'node_uuid': node.uuid, 'error': exc})
-             raise exception.HardwareInspectionFailure(error=exc)
- 
-+        pxe_dev_nics = self._get_pxe_dev_nics(client, nics, node)
-+        if pxe_dev_nics is None:
-+            LOG.warning('No PXE enabled NIC was found for node '
-+                        '%(node_uuid)s.', {'node_uuid': node.uuid})
-+        
+--- inspect.py	2019-03-27 09:53:02.035769407 +0000
++++ inspect_modified.py	2019-03-27 09:51:32.965097867 +0000
+@@ -122,8 +122,10 @@
          for nic in nics:
              try:
                  port = objects.Port(task.context, address=nic.mac,
 -                                    node_id=node.id)
 +                                    node_id=node.id,
 +                                    pxe_enabled=(nic.id in pxe_dev_nics))
```

Comment:
@kpradip001  Above lines(line no:7-11) should be remain in patch

Reply:
I believe this patch is missing a section that sets:
pxe_dev_nics = self._get_pxe_dev_nics(...)

Reply:
updated this in latest commit